### PR TITLE
frontend structure done

### PR DESCRIPTION
**Pull request by sanjuel**
navlink changed out for standard a href tag. Navlink wouldnt work, but when we changed it back to what we knew works, it worked.

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { NavLink } from "react-router-dom"
```
Comment (SunkissedQueen) [inline src/components/Header.js]: If not using data remove it Navlink from react-router-dom. Not sure what your syntax was on NavLink but check out this site https://v5.reactrouter.com/web/api/NavLink/isactive-func

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Change request: 
- Don't see assets folder
- On App.js, Need to import the array stored on MockCat.js, look at your casing

For future PRs:
I am expecting to see these changes as part of the next PR
- If not using data remove it, for instance, Navlink from react-router-dom. I also included a link to see if your syntax was off.
- Make sure opening and closing tags are lined up with proper indentation

File: src/components/Header.js (hunk 1 above)
Comment (JPD99) [reply src/components/Header.js]: The assets folder is under the src directory from the original push. And when importing the mockCat.js file with the variable mockCat, I believe you wish for us to display the entire array in the display portion of App.js , is that correct? Thank you in advance!

Comment (JPD99) [reply src/components/Header.js]: Oh wait! The assets folder is not in the original push, sorry for that false statement.

**Review by SunkissedQueen — APPROVED** [review]
Thanks for making the changes. For clarity on indentation, here's an example.
```javascript
const App = () => {
  return (
    <>
      <Header />
      <Routes>
        <Route path="/" element={<Home />} />
        <Route path="/catindex" element={<CatIndex />} />
        <Route path="/catshow" element={<CatShow />} />
        <Route path="/catnew" element={<CatNew />} />
        <Route path="/catedit" element={<CatEdit />} />
        <Route path="*" element={<NotFound />} />
      </Routes>
      <Footer/>
    </>
  );
}
export default App;
```
We can talk further about indentation later. If this example is not helpful

**Pull request merged**